docs/content/developers/building-contributing.md

```diff
@@ -1,41 +1,43 @@
+To download the latest version, you can visit the [master-build](https://github.com/drud/ddev/actions/workflows/master-build.yml) workflow and choose the latest item (or the one that matches a commit you want to test). The artifacts for each OS are at the bottom of the page.
 
-And of course you can just see the latest build in action by visiting ddev on [gitpod](https://gitpod.io/#https://github.com/drud/ddev).
+Alternateively, you can see the latest build in action by visiting DDEV on [Gitpod](https://gitpod.io/#https://github.com/drud/ddev).
 
 ## Testing a PR
 
-Each build of a PR has artifacts created in github, so you can click the details of the [PR Build](https://github.com/drud/ddev/actions/workflows/pr-build.yml) test, choose the PR you want to work with, and download the artifacts you need there.
+Each build of a PR has artifacts created in GitHub, so you can click the details of the [PR Build](https://github.com/drud/ddev/actions/workflows/pr-build.yml) test, choose the PR you want to work with, and download the artifacts you need.
 
-After you download and unzip the appropriate binary, you can place it in your $PATH. The easiest way to do this if you're using homebrew is `brew unlink ddev` and then `unzip ddev.zip && chmod +x ddev && mv ddev /usr/local/bin/ddev`.
+After you download and unzip the appropriate binary, you can place it in your `$PATH`. The easiest way to do this if you’re using Homebrew is `brew unlink ddev` and then `unzip ddev.zip && chmod +x ddev && mv ddev /usr/local/bin/ddev`.
 
-You can verify that the replacement worked via `ddev -v`. The output should be something like `ddev version v1.19.1-42-g5334d3c1` (instead of the regular `ddev version v1.19.1`).
+Verify the replacement worked via `ddev -v`. The output should be something like `ddev version v1.19.1-42-g5334d3c1`, instead of the regular `ddev version v1.19.1`.
 
-(On macOS Big Sur these downloaded binaries are not signed, so you will want to `xattr -r -d com.apple.quarantine /path/to/ddev` in order to use them. The binaries on the master branch and the final binaries in any release are signed, of course.)
+(On macOS Big Sur these downloaded binaries are not signed, so you’ll want to `xattr -r -d com.apple.quarantine /path/to/ddev` in order to use them. The binaries on the master branch and the final binaries in any release *are* signed.)
 
-You do not typically have to install anything else other than the downloaded binary; when you run it it will access any docker images that it needs.
+You do not typically have to install anything else other than the downloaded binary; when you run it it will access any Docker images that it needs.
 
-After you're done, you can remove the downloaded binary and `brew link ddev`.
+After you’re done, you can remove the downloaded binary and `brew link ddev`.
```

Comment:
Please don't use anything but real apostrophes. Other punctuation causes havoc with text editors and othere things.

Reply:
Curly quotes [are more typographically correct](https://practicaltypography.com/straight-and-curly-quotes.html), which is why I called that out as an item to fix in the PR goals—since it’s specifically documentation in Markdown.